OneCommunityGlobal/HGNRest
# Carl fix suggestion options

# Description
(PRIORITY: HIGH): Suggestion Icon does not return options for users to select what they want.

## Related PRS (if any):
This backend PR is related to the [#1292](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1292) frontend PR.
To test this backend PR you need to checkout the [#1292](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1292)  frontend PR.
…

## Main changes explained:
- Delete SuggestionModalData.json file
- Created a suggestionData variable with the json data locally
- Changed how the various functions accessed the data
…

## How to test:
1. check into current branch
2. I will suggest go to dashboardController.js-> sendMakeSuggestion function -> emailSender -> change the email address from oneCommunity to your gmail. So that you can easily verify that email successfully send.
3. follow this link https://docs.google.com/document/d/1j92U7Vr14gQLehr1e_gkMdqWcOA37WGxNLv9yTa96CY/edit and Aishwarya Kalkundrikar's comment on how to send email
4. do `npm install` and `...` to run this PR locally
5. log as admin/owner user
6. go to dashboard→ Suggestion Icon -> Make Edits, Add/Delete

###  Development
[screencast-whereby.com-2023.09.27-14_51_45.webm](https://github.com/OneCommunityGlobal/HGNRest/assets/68056104/f6974739-a1f1-498e-9139-bcda5edd6bca)

### Current branch
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.09.27-14_27_28.webm](https://github.com/OneCommunityGlobal/HGNRest/assets/68056104/c66814c2-23d5-4651-9681-07e4c88c0523)

###  Email

![image](https://github.com/OneCommunityGlobal/HGNRest/assets/68056104/f546a394-82ba-4c49-a9c4-4a2e98854456)


## Screenshots or videos of changes:
